community-edition/readme.md

```diff
@@ -1,48 +1,57 @@
+# Introduction
+
+Community Edition is a free tool designed to help developers deploy the full Hubs stack on cloud computing software of their choosing. Community Edition simplifies and automates most of the complex deployment process using Kubernetes, a containerized software orchestration system.
+
+Community Edition is designed for developers capable of working with the full Hubs stack and of navigating complex application infrastructure. With Community Edition, developers are responsible for designing, building, hosting, and maintaining their software throughout its lifetime. In order to create a production-ready version of Hubs similar to Hubs Cloud, Developers using Community Edition will need to implement additional features and customizations outside of those listed in this repo. See "Considerations for Production Environment".
+
+# Why Kubernetes
+
+Kubernetes is an industry standard for allowing developers to build, deploy, and scale application efficiently and reliably. Benefits to Kubernetes include:
```

Comment:
> Kubernetes is an industry standard for allowing developers to build...

i don't think k8s is that, it's an free opensource container orchestrator
it doesn't host code directly so it's technically incorrect to say that it's for developers to build/deploy/scale -- serverless options on public clouds (ie aws-lambda) are the tools designed and marketed for that usage

Reply:
Would it be better to say "assisting developers as they build, deploy, and scale..."?